patches/v8/build_remove_legacy_oom_error_callback.patch

```diff
@@ -0,0 +1,168 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: VerteDinde <keeleymhammond@gmail.com>
+Date: Tue, 14 Jun 2022 15:12:43 -0700
+Subject: build: remove legacy oom error callback
+
+Addresses 3646014: [API] Deprecate LegacyOOMErrorCallback:
+https://chromium-review.googlesource.com/c/v8/v8/+/3646014
+
+This deprecation was added to warn embedders that the legacy method
+was being replaced. However, Electron does not use the legacy method.
+The deprecation causes build issues on Windows, this patch removes
+the unused method entirely to avoid those errors.
```

Comment:
Would be good to add, for future context: the reason these methods cause issues for us is that we export the methods, which counts as "usage" of the deprecated stuff!

Reply:
Thanks, that's a good callout! I'll update the patch description 🙂 Do you think it's already to go ahead and remove these methods, and just call out the change in the release notes in case other apps are using the old exported methods? From what I could tell in the bug description, both methods do essentially the same thing, and they weren't expecting the switch over to cause breaking issues

Reply:
Yeah, I think it's okay to remove the methods. There's a clear upgrade path.